.github/workflows/check-commit-message.yml

```
@@ -0,0 +1,12 @@
+﻿name: Check Commit Message
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)